chromatic/tests/test_summaries.py

```diff
@@ -7,3 +7,63 @@ def test_lightcurve_and_spectrum_summaries():
     s = SimulatedRainbow().inject_transit()
     ax[0].plot(s.time, s.get_lightcurve())
     ax[1].plot(s.wavelength, s.get_spectrum())
+
+
+def test_measured_scatter_summary():
```

Comment:
I added this test to see how it works! This should run automatically every time you run `pytest` from the repository directory, but you should also be able to run it from any notebook with:
```python
from chromatic.tests import *
test_measured_scatter_summary()
```
It should spit out a plot into an `examples/` directory that looks like this:
![measured-scatter-demo](https://user-images.githubusercontent.com/4691081/173421685-b5f45aa5-8cd5-465f-902d-6ee362d68861.png)

This both makes sure that the code is giving answers that are approximately quantitatively correct and makes a plot comparing the methods. It looks like `'MAD'` gives a slightly noisier estimate of the scatter than `standard-deviation` (but it handles outliers way better, so that's probably worth it!)